src/platform/ESP32/bluedroid/BLEManagerImpl.cpp

```diff
@@ -610,7 +1117,7 @@ CHIP_ERROR BLEManagerImpl::InitESPBleLayer(void)
     }
 
     // Set the maximum supported MTU size.
-    err = MapBLEError(esp_ble_gatt_set_local_mtu(ESP_GATT_MAX_MTU_SIZE));
+    err = MapBLEError(esp_ble_gatt_set_local_mtu(256));
```

Comment:
If we are hardcoding this, can be define it or declare a const for this?

Reply:
yes, ok